addons/mail/views/mail_activity_views.xml

```diff
@@ -332,4 +344,85 @@
         </field>
     </record>
 
+    <record id="mail_activity_open_target_view_tree" model="ir.ui.view">
```

Comment:
:socks: always begin with model information in naming, then suffix to distinguish them (see guidelines)
```suggestion
    <record id="mail_activity_view_tree_open_target" model="ir.ui.view">
```